lib/nexpose/scan_template.rb

```diff
@@ -583,5 +583,20 @@ def enable_enhanced_logging=(enable)
       self.enable_debug_logging = enable
       self.aces_level = (enable ? 'full' : 'none')
     end
+
+    # Enable or disable windows service editor
+    # @param [Boolean] enable Enable or disable windows service editor
```

Comment:
Where's the typo sorry?
Not sure if getter would be ever used, so did not add it.